ft_prepare_sourcemodel.m

```diff
@@ -668,6 +668,9 @@
 end
 
 if ~isempty(cfg.moveinward)
+  if ~any([strcmp(cfg.method, 'basedonshape'), strcmp(cfg.method, 'basedoncortex'), strcmp(cfg.method, 'basedonvol'), strcmp(cfg.method, 'basedonfile')])
+      ft_warning('cfg.moveinward is designed to work with surface sourcemodels, not 3D grids.')
```

Comment:
I am not sure how to handle `'basedonfile'` here, I assume it can result in any other kind of method down the road, depending on what is loaded?

Reply:
I think that it is ok (like you do) not give a warning when `basedonfile`